db/migrate/20161127084744_remove_featuredfrom_skins.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveFeaturedfromSkins < ActiveRecord::Migration
```

Comment:
This migration is currently not working because it's not in Rails 5 format.

```
Caused by:
StandardError: Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:
  class RemoveFeaturedfromSkins < ActiveRecord::Migration[4.2]
```